app/services/payout_report_publisher_includer.rb

```diff
@@ -73,16 +73,16 @@ def perform
 
     # Notify publishers that have money waiting, but will not will not receive funds
     if total_probi > PROBI_THRESHOLD && @should_send_notifications
-      send_emails(uphold_connection)
+      send_emails(uphold_connection, probi_to_bat(total_amount))
```

Comment:
Where's `total_amount` coming from?

Reply:
good catch, should have been total_probi!